### DISCUSSION TITLE
Intermediate Images are tiny and video not being made

### DISCUSSION THREAD

**Discussion by emilbigaj** [AlUlkesh/sd_save_intermediate_images#15] [General]
I've been able to save intermediate images but they're really small. Im generating 512x512 images but the intermediate images end up being something like 5% that size. Am I missing something?

Secondly the videos isnt being made. Whats the filepath that the video should be saved?

This is what my setting look like.

![image](https://user-images.githubusercontent.com/18477294/213063399-71b680dc-c25a-4d80-b537-53c6ce5ac8a0.png)

any help much appreciated!

**Comment by AlUlkesh** [comment]
Yes, you need to set this to "Full" in the settings for full size intermediates:

![image](https://user-images.githubusercontent.com/99896447/213106804-2b5922f4-7f9d-4c61-9c51-f4430ed1c7b2.png)

**Comment by sx2astra** [comment]
` File "D:\Git\stable-diffusion-webui\venv\lib\site-packages\ffmpy.py", line 98, in run
    raise FFExecutableNotFoundError(
ffmpy.FFExecutableNotFoundError: Executable 'ffmpeg' not found`

Where do you put ffmpeg executable for windows?

**Reply by AlUlkesh** [reply]
Easiest way would probably be to download one of the release packages:
https://www.gyan.dev/ffmpeg/builds/#release-builds

unpack it whereever you want and then put the path to the bin folder in your path environment variable.
